### PR TITLE
EVG-19807: Fix form validation for container size definitions

### PR DESCRIPTION
**Pull request by minnakt**
EVG-19807

### Description
The validation for container size definitions seems to not be working properly, as I'm able to save the changes when memory and CPU are empty. This PR updates the form schema so that it works correctly.

### Screenshots
Before:
Screenshot 2023-05-05 at 12 29 46 PM

After:
Screenshot 2023-05-05 at 12 30 23 PM

### Testing
- Tried to fix the flaky tests, which I ran 5 times on CI and it passed all times so I am hopeful 🤞

### CODE REVIEW

File: cypress/integration/projectSettings/project_settings.ts [hunk 1]
```diff
@@ -1112,8 +1114,7 @@ describe("Containers", () => {
   });
   it("should be able to add a container configuration and save it", () => {
     cy.dataCy("add-button").should("be.visible");
-    cy.dataCy("add-button").click();
-
+    cy.dataCy("add-button").trigger("mouseover").click();
```
Comment (khelif96) [inline cypress/integration/projectSettings/project_settings.ts:1117]: I wish I knew why just clicking on it is not enough.

**Review by khelif96 — APPROVED** [review]
Queen 👑 thanks for fixing this

**Review by khelif96 — APPROVED** [review]
Queen 👑 thanks for fixing this

File: cypress/integration/projectSettings/project_settings.ts (hunk 1 above)
Comment (minnakt) [reply cypress/integration/projectSettings/project_settings.ts:1117]: honestly it is a little hack... 😭 but I can't replicate the failure on local, only on CI rip

**Pull request merged**